js/main.js

```diff
@@ -309,38 +354,35 @@ window.addEventListener('click', goBack);
 
 function switchList(event) {
   if (event.target.getAttribute('id') === 'right' || event.target.getAttribute('id') === 'right-arrow') {
-    removeChildren($ul);
-    count++;
-    var current = determineView() + count;
-    getAPIData(current);
-  } if (event.target.getAttribute('id') === 'left' || event.target.getAttribute('id') === 'left-arrow') {
-    removeChildren($ul);
-    count--;
-    current = determineView() + count;
-    getAPIData(current);
+    if (count !== 44) {
+      count += 22;
+    }
   }
-}
-
-function determineView() {
-  if ($ajaxList.getAttribute('data-view') === 'character') {
-    return 'character/?page=';
-  } else if ($ajaxList.getAttribute('data-view') === 'location') {
-    return 'location/?page=';
-  } else if ($ajaxList.getAttribute('data-view') === 'episode') {
-    return 'episode/?page=';
+  if (event.target.getAttribute('id') === 'left' || event.target.getAttribute('id') === 'left-arrow') {
+    if (count !== 0) {
+      count -= 22;
+    }
+  }
+  for (var i = 0; i < $ul.children.length; i++) {
+    if (count === 0) {
+      $ul.children[0].className = 'list-item';
+    }
+    if (i < count || i >= (count + 22)) {
```

Comment:
What is the meaning of `22` here? Is there a property value you could based this off of instead of an arbitrary number?